.github/workflows/json.yml

```diff
@@ -0,0 +1,66 @@
+name: Json
+
+on: [push, pull_request]
+
+jobs:
+  pass:
+    name: >-
+      pass ${{ matrix.os }} ${{ matrix.ruby }}
+    runs-on: ${{ matrix.os }}-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu, macos, windows ]
```

Comment:
Do not mix Linux/macOS and Windows platforms in the one workflow.

Reply:
I am curious, why do you prefer separate workflows?
I usually prefer like this if there isn't much OS-specific logic, as it avoids duplication and makes it significantly easier to maintain (1 place to change vs 3).

Reply:
The maintainability is low about mixing *nix and Windows for my experience.

Reply:
That may have been true in the past at the beginning of GitHub Actions, but I don't think it's the case anymore, especially with improvements in ruby/setup-ruby. In fact many gems already use such an approach and in my experience, it's successful.

From a maintainability point of view it certainly looks better to me to have the steps repeated the minimum number of times (2 times here for allowed failures, instead of 3-4 if we have one job definition per OS).